string info
uint64 time_stamp
int8 servo_control
